charts/thoras/values.yaml

```diff
@@ -95,3 +95,6 @@ thorasMonitor:
 
 thorasForecast:
   requestCpu:
+
+thorasAgent:
+  enabled: false
```

Comment:
Not enabled by default. This will likely be enabled once we have the basic, beta-version completed.